.github/workflows/pr-actions.yml

```diff
@@ -48,37 +48,61 @@ jobs:
   # Run Jest Unit Tests
   test-unit:
     uses: ./.github/workflows/run-npm-script.yml
-    needs: lint
+    needs: draft
     with:
       label: 'Test: Unit'
       script: 'npm run test:unit'
 
   # Run Cypress End to End Tests
-  test-e2e:
-    name: 'Test: End to End'
-    runs-on: ubuntu-latest
-    needs: test-unit
-    # stop the job if it runs over 20 minutes to prevent a hanging process from eating minutes
-    timeout-minutes: 45
+  install:
+    needs: draft
+    runs-on: ubuntu-22.04
     steps:
-      # Checkout and node setup are shared in both `pr-actions.yml` and `run-npm-script.yml`
-      # If you change one, please change the other
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cypress install
+        uses: cypress-io/github-action@v5
         with:
-          node-version: '18.12.1'
-          cache: 'npm'
-      # https://github.com/cypress-io/github-action
-      - uses: cypress-io/github-action@v4
+          # Disable running of tests within install job
+          runTests: false
+          build: npm run build
+
+      - name: Save build folder
+        uses: actions/upload-artifact@v3
+        with:
+          name: build
+          if-no-files-found: error
+          path: assets
```

Comment:
This is the secret sauce that allows the parallel containers to share the single build of the frontend without repeating the build step